barretenberg/cpp/src/barretenberg/protogalaxy/combiner.test.cpp

```diff
@@ -52,7 +52,7 @@ TEST(Protogalaxy, CombinerOn2Instances)
             }
 
             ProverInstances instances{ instance_data };
-            instances.alphas.fill(Univariate<FF, 13>(FF(0))); // focus on the arithmetic relation only
+            instances.alphas.fill(bb::Univariate<FF, 13>(FF(0))); // focus on the arithmetic relation only
```

Comment:
I'm surprised this `bb::` is necessary because it's only one level up from `bb::honk` which is `using`ed in this file.